cypress/integration/tx_modal.spec.js

```diff
@@ -0,0 +1,192 @@
+const TEST_SAFE = 'rin:0x11Df0fa87b30080d59eba632570f620e37f2a8f7'
+const RECIPIENT_ENS = 'diogo.eth'
+const RECIPIENT_ADDRESS = '0x6a5602335a878ADDCa4BF63a050E34946B56B5bC'
+const SAFE_NONCE = '5'
+
+describe('Tx Modal', () => {
+  before(() => {
+    cy.connectE2EWallet()
+
+    // Open the Safe used for testing
+    cy.visit(`/${TEST_SAFE}`)
+    cy.contains('a', 'Accept selection').click()
+  })
+
+  describe('Send funds modal', () => {
+    it('should open the modal when clicking New Transaction', () => {
+      // Modal should be closed
+      cy.get('[aria-describedby="Send Tokens Form"]').should('not.exist')
+
+      // Opens Tx Modal
+      cy.contains('New Transaction').click()
+      cy.get('[aria-describedby="Send Tokens Form"]').should('be.visible')
+      cy.get('.smaller-modal-window').contains('Send')
+    })
+
+    it('should close the modal by clicking outside of the modal', () => {
+      // Modal is open
+      cy.get('.smaller-modal-window').contains('Send')
+
+      // Click outside of the .smaller-modal-window to close the modal
+      cy.get('.smaller-modal-window').then(($target) => {
+        let coords = $target[0].getBoundingClientRect()
+        const { x, y } = coords
+        cy.get('body').click(x - 10, y - 10)
+      })
```

Comment:
We don't need to test trivial things like that.

Reply:
I was thinking if we decide to change this behaviour (clicking on the background closes the modal) it would be caught by this test. But I agree it's not testing anything worthy.